src/renderers/WebGLRenderer.js

```diff
@@ -1554,6 +1554,7 @@ THREE.WebGLRenderer = function ( parameters ) {
 		if ( material instanceof THREE.MeshPhongMaterial ||
 				material instanceof THREE.MeshLambertMaterial ||
 				material instanceof THREE.MeshStandardMaterial ||
+				material instanceof THREE.MeshPhysicalMaterial ||
```

Comment:
These checks aren't needed because `THREE.MeshPhysicalMaterial` extends `THREE.MeshStandardMaterial` so the check is already true.